src/next/str.rs

```diff
@@ -21,12 +21,13 @@
 //#
 //# ## Other
 //# - ClaimableBalanceId
+//# - PoolId
 #![cfg(feature = "alloc")]
 
 use super::{
     AccountId, AssetCode, AssetCode12, AssetCode4, ClaimableBalanceId, Error, Hash, MuxedAccount,
-    MuxedAccountMed25519, NodeId, PublicKey, ScAddress, SignerKey, SignerKeyEd25519SignedPayload,
-    Uint256,
+    MuxedAccountMed25519, MuxedEd25519Account, NodeId, PoolId, PublicKey, ScAddress, SignerKey,
```

Comment:
Can you add a Display and FromStr impl for the `MuxedEd25519Account` type? You'll need to include the type in the `Makefile`s list of types to exclude the derived variety for. Same for PoolId and ClaimableBalanceId when they're implemented.

Reply:
Done. I've also updated the makefile, though I'm not sure what exactly this does.